src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/config/ResourceServerConfiguration.kt

```diff
@@ -1,18 +1,21 @@
 package uk.gov.justice.digital.hmpps.visitscheduler.config
 
+import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 import org.springframework.security.config.annotation.method.configuration.EnableGlobalMethodSecurity
 import org.springframework.security.config.annotation.web.builders.HttpSecurity
 import org.springframework.security.config.annotation.web.configuration.EnableWebSecurity
-import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter
 import org.springframework.security.config.http.SessionCreationPolicy
+import org.springframework.security.web.SecurityFilterChain
 
 @Configuration
 @EnableWebSecurity
 @EnableGlobalMethodSecurity(prePostEnabled = true, proxyTargetClass = true)
-class ResourceServerConfiguration : WebSecurityConfigurerAdapter() {
```

Comment:
WebSecurityConfigurerAdapter is an old way of doing things it is now deprecated and needs to be changed migrated into the @Bean approach